ftp.go

```diff
@@ -564,6 +531,36 @@ func (c *ServerConn) List(path string) (entries []*Entry, err error) {
 	return
 }
 
+// SetUTF8 issues an "OPTS UTF8 ON" command.
```

Comment:
Can you rename the function in-place to no mess up with git history ?

Reply:
Ofc, done.